setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='rapidpro_flow_tools',
+    version='2.0.7',
```

Comment:
Comes from me running into bugs and arbitrarily increasing the version number when trying to fix them! Appreciate is very bad practice as should have done more local testing and only uploaded once I was sure it was bug free but there was lots of small things I missed initially.

Reply:
That's fine. What we're saying is whatever came before was v1, and this PR takes us to v2. We're currently at 2.0.7 because you've released some patches in the course of preparing this PR.

Let's lock the current version down at 2.0.7. If you need to do something like a pre-release, use the following notation: 2.0.8-rc.1, 2.0.8-rc.2, ... , 2.0.8-rc._n_. 'rc' meaning release candidate. When ready for an official release remove the '-rc._n_' bit - e.g. 2.0.8.